vsts/node-release.yaml

```diff
@@ -0,0 +1,185 @@
+
+      if ($(Test-Path -Path $configFile -PathType Leaf) -ne $true) {
+          throw "File not found: $($configFile)"
+      }
+
+      if ($(Test-Path -Path $app -PathType Leaf) -ne $true) {
+          throw "File not found: $($app)"
+      }
+
+      node $app $configFile "$(Build.SourcesDirectory)"
+      
+    displayName: Bump Version
+    enabled: true
+
+  - script: dir /b /s
+
+  - powershell: |
+      $releaseTag = Get-Date -UFormat "%Y-%m-%d"
+      $releaseBranch = "release_$($releaseTag)"
+
+      Set-Location "$(Agent.BuildDirectory)\s"
+
+      git config user.email $(GitHubEmail)
+      git config user.name $(GitHubName) 
+      git checkout -b $releaseBranch
+      git add .
+      git commit -m "Bump package versions for release $($releaseTag)"
+      git tag -d $($releaseTag)
+      git tag -a $($releaseTag) -m "SDK release on $($releaseTag)"
+      git push --tags -u origin $($releaseBranch)
+      git checkout master
+      git branch -d $releaseBranch
+      
+    displayName: Create Release Branch
+    enabled: true
+
+  - powershell: |
+       runas.exe /savecred /user:administrator
+       npm install --global node-gyp
+       npm install --global lerna
+    displayName: 'Install Dependencies'
+    enabled: false
+
+  - script: |
+       call lerna bootstrap
+       call lerna run build
+    displayName: 'Bootstrap & Build'
+    enabled: false
+
+  - script: 'call lerna run ci'
+    displayName: 'Unit & Integration Tests'
+    env:
+      IOTHUB_CONNECTION_STRING: $(WINDOWS-IOTHUB-CONNECTION-STRING)
+    enabled: false
+
+  - script: 'lerna run e2e'
+    displayName: 'E2E Tests'
+    env:
+      IOTHUB_CA_ROOT_CERT: $(IOTHUB-CA-ROOT-CERT)
+      IOTHUB_CA_ROOT_CERT_KEY: $(IOTHUB-CA-ROOT-CERT-KEY)
+      IOT_PROVISIONING_DEVICE_ENDPOINT: $(IOT-PROVISIONING-DEVICE-ENDPOINT)
+      IOTHUB_CONNECTION_STRING: $(WINDOWS-IOTHUB-CONNECTION-STRING)
+      STORAGE_CONNECTION_STRING: $(WINDOWS-STORAGE-CONNECTION-STRING)
+      IOT_PROVISIONING_DEVICE_IDSCOPE: $(WINDOWS-IOT-PROVISIONING-DEVICE-IDSCOPE)
+      IOT_PROVISIONING_ROOT_CERT: $(WINDOWS-IOT-PROVISIONING-ROOT-CERT)
+      IOT_PROVISIONING_ROOT_CERT_KEY: $(WINDOWS-IOT-PROVISIONING-ROOT-CERT-KEY)
+      IOT_PROVISIONING_SERVICE_CONNECTION_STRING: $(WINDOWS-IOT-PROVISIONING-SERVICE-CONNECTION-STRING)
+      DPS_CONN_STRING_INVALID_CERT: $(DPS-CONN-STRING-INVALID-CERT)
+      DPS_GLOBAL_DEVICE_ENDPOINT_INVALID_CERT: $(DPS-GLOBAL-DEVICE-ENDPOINT-INVALID-CERT)
+      IOTHUB_CONN_STRING_INVALID_CERT: $(IOTHUB-CONN-STRING-INVALID-CERT)
+      IOTHUB_DEVICE_CONN_STRING_INVALID_CERT: $(IOTHUB-DEVICE-CONN-STRING-INVALID-CERT)
+    enabled: false
+
+  - task: PublishTestResults@2
+    displayName: 'Publish Test Results | Mocha'
+    inputs:
+      testResultsFiles: 'e2etests/test-results.*.xml'
+      mergeTestResults: true
+      testRunTitle: 'E2E Tests - Windows'
+    condition: succeededOrFailed()
+    enabled: false
```

Comment:
I keep going back & forth on this - given we're going to build the release branch with a build triggered by the PR, I don't think we need to run the tests here. change my mind?